WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -220,6 +220,21 @@ protected SendControlViewModel(WalletService walletService, string title)
 					}
 				});
 
+			// Triggering the detection of same address values.
+			this.WhenAnyValue(x => x.Address)
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(_ =>
```

Comment:
No need for the braces...

`.Subscribe(_ => this.RaisePropertyChanged(nameof(Address));`